### PR TITLE
use generic `bee` binary name

### PR DESCRIPTION
**Pull request by lgadban**
Update the `bee package` command to work with the newly renamed `bee` binary (which was done in https://github.com/solo-io/bumblebee/pull/80)

### CODE REVIEW

File: pkg/cli/internal/commands/package/Dockerfile [hunk 1]
```diff
@@ -7,4 +7,4 @@ COPY ./store /root/.bumblebee/store/
 
 ARG BPF_IMAGE
 ENV BPF_IMAGE=$BPF_IMAGE
-CMD ./bee-linux-amd64 run --no-tty ${BPF_IMAGE}
+CMD ./bee run --no-tty ${BPF_IMAGE}
```
Comment (EItanya) [inline pkg/cli/internal/commands/package/Dockerfile:10]: Are we not using this `make` target? https://github.com/solo-io/bumblebee/blob/dfcddf413a7cfadea574438783945b535685f4dc/Makefile#L78

Comment (lgadban) [reply pkg/cli/internal/commands/package/Dockerfile:10]: I don't think so -- I don't think we ever got arm wired up and working correctly

Comment (lgadban) [reply pkg/cli/internal/commands/package/Dockerfile:10]: but to add some more context, post-build when adding the built binary into the image, we are changing the name to be generic (at least for now)

https://github.com/solo-io/bumblebee/blob/main/bee/Dockerfile-bee#L8

Comment (EItanya) [reply pkg/cli/internal/commands/package/Dockerfile:10]: Sorry, didn't mean ARM specifically, just meant the target which creates a suffixed binary. Yea your second comment answers the question!

**Review by EItanya — APPROVED** [review]

**Pull request merged**